src/index.ts

```diff
@@ -36,6 +36,7 @@ export {
   getSwaggerFiles,
   SwaggerFileList,
   IService,
+  getDefaultTag,
```

Comment:
Removed in this PR, I suspect accidentally:

https://github.com/Azure/avocado/pull/93/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80L203